src/celestia/qt/qtappwin.cpp

```diff
@@ -251,9 +254,9 @@ CelestiaAppWindow::init(const CelestiaCommandLineOptions& options)
     if (!logPath.makeAbsolute())
         QMessageBox::warning(0, "Celestia", _("Error getting path for log filename!"));
 
-    QString celestia_data_dir = options.startDirectory.isEmpty()
-        ? QString::fromLocal8Bit(::getenv("CELESTIA_DATA_DIR"))
-        : options.startDirectory;
+    QString celestia_data_dir = options.startDirectory;
```

Comment:
As you're touching this code, could you rename `celestia_data_dir` to `celestiaDataDir` so it matches the file style?